config.yaml

```diff
@@ -118,3 +118,10 @@ communications:
   slack:
     channel: 'SLACK_CHANNEL'
     token: 'SLACK_API_TOKEN'
+
+# Setting to support multiple clusters
+settings:
+  # Cluster name to differentiate incoming messages
+  clustername: dev-cluster
```

Comment:
Should this be part of helm installation? In the way we accept slack token.

Default can be `not-configured` or `unknown`